dotnet/Microsoft.iOS.Sdk/data/UnixFilePermissions.xml

```diff
@@ -5,4 +5,7 @@
 	<!-- mlaunch -->
 	<File Path="tools/bin/mlaunch" Permission="755" />
 	<File Path="tools/lib/mlaunch/mlaunch.app/Contents/MacOS/mlaunch" Permission="755" />
+     <!-- xcsync -->
+	<File Path="tools/bin/xcsync" Permission="755" />
+	<File Path="tools/lib/xcsync" Permission="755" />
```

Comment:
How much bigger does the iOS SDK pack get when bundling xcsync? It might make more sense to ship this as a separate sdk or tools pack within the workload.

Reply:
~150MB / workload, we had a discussion with @jonathanpeppers on how to go about doing that. Was planning on discussing with you once you were back. And look, you're back!

Reply:
Ugh, that's a lot. Let's talk on Monday's call to see if we can avoid that.

Reply:
@dalexsoto seems 'ok' with it and fix it in the next preview

Reply:
Discussed during the weekly call, this will be enhanced in the next Preview.